hub/route/server.go

```diff
@@ -342,11 +358,26 @@ func getLogs(w http.ResponseWriter, r *http.Request) {
 		}
 		buf.Reset()
 
-		if err := json.NewEncoder(buf).Encode(Log{
-			Type:    logM.Type(),
-			Payload: logM.Payload,
-		}); err != nil {
-			break
+		if !isStructured {
+			if err := json.NewEncoder(buf).Encode(Log{
+				Type:    logM.Type(),
+				Payload: logM.Payload,
+			}); err != nil {
+				break
+			}
+		} else {
+			newLevel := logM.Type()
+			if newLevel == "warning" {
+				newLevel = "warn"
+			}
+			if err := json.NewEncoder(buf).Encode(LogStructured{
+				Time:    time.Now().Format("15:04:05"),
```

Comment:
```suggestion
				Time:    time.Now().Format("00:00:00"),
```

Reply:
That's go's time format string. You can't change it.